Library/Homebrew/extend/os/linux/system_config.rb

```diff
@@ -42,8 +42,9 @@ def host_ruby_version
 
     def dump_verbose_config(out = $stdout)
       dump_generic_verbose_config(out)
-      out.puts "Kernel: #{`uname -mors`.chomp}"
+      out.puts "Kernel: #{Utils.safe_popen_read("uname", "-mors").chomp}"
       out.puts "OS: #{OS::Linux.os_version}"
+      out.puts "WSL: #{Utils.safe_popen_read("wsl", "--version").chomp}" if which("wsl")
```

Comment:
`wsl --version` is not available within WSL - it's only available in a Windows command prompt.